Dockerfile

```diff
@@ -13,12 +13,12 @@ RUN apt-get update \
 SHELL ["/bin/bash", "--login", "-c"]
 
 RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
-RUN nvm install lts/*
```

Comment:
@guarani I am just realizing that [this step](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/.buildkite/pipeline.yml#L23) that I linked in my previous comment is not correct or at least it wasn't 🤦  I think it needed to be:

```
pushd gutenberg && nvm install && nvm use && popd
```

It looks like you added the `.nvmrc` file [in this commit](https://github.com/wordpress-mobile/gutenberg-mobile/commit/596814ea002125076028087a5b92992a8f539001) so it's correct now 😅 

---

Umm.. while compiling this comment, I found [this commit](https://github.com/wordpress-mobile/gutenberg-mobile/commit/75cbe43f07d43574e37e1de3d65f98887872e2ea) so it was correct before, you just changed it with the recent addition of `.nvmrc` which makes sense. I thought I made a mistake before.

That was an unnecessary roller-coaster of emotions 😂

Reply:
😄 It sounds like all is good then with regard to this change, is that correct?

Reply:
I think my initial comment is still valid. You can ignore the second comment, it was mostly sharing the roller-coaster, it doesn't affect my first comment.

Reply:
Sorry, I'm not sure I understand your original comment. What specific changes do you propose?

In the current setup, why is `RUN nvm install lts/*` used instead of cd'ing into the gutenberg-mobile folder and running `nvm install && nvm use`?

Reply:
Because the setup was intended to be generic as our decision here will only impact caching. The correct version should always be installed by the project itself.

The current iteration of this Docker image is:
* We cache `.npm` for gutenberg-mobile dependencies
* We re-create the `latest` tag nightly

Neither of these are guaranteed to stay the same, we could possibly switch to caching in S3 instead of the Docker image. So, if we removed the block that does the current caching, it'd leave the Docker image with no `nvm` cached.

There is an argument to be made for keeping the `Dockerfile` bare, and only have what's necessary. And, I am typically in favor of that. However, this project is maintained by Platform 9 and not everyone in the team has experience with nvm/npm, so I could imagine a case where the caching block is removed without realizing that we need to re-add nvm caching.

When we have both these lines, the only impact it'll have is that while building the image the second `nvm install` command in 99% of the time will say that this version is already installed.

Does that make sense?

Reply:
Yes, I think I follow now: the Dockerfile itself should be independent of the [caching block](https://github.com/wordpress-mobile/docker-gb-mobile-image/blob/f567740a5eb432adb2588de5476ab7a67b093d7b/Dockerfile#L18-L24), so you prefer we kept the current `RUN nvm install lts/*` [here](https://github.com/wordpress-mobile/docker-gb-mobile-image/blob/f567740a5eb432adb2588de5476ab7a67b093d7b/Dockerfile#L16) _and_ also add the [`nvm install`](https://github.com/wordpress-mobile/docker-gb-mobile-image/pull/2/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R21)?

Reply:
Yeap, it looks good 👍

Reply:
Awesome, thanks for the review!